Update docs to replace `createInteractor` in favor of `extends` (#921)
thefrontside/bigtest: website/docs/interactors/3-locators-filters-actions.md

</label>
```

> _See the source code for the TextField interactor [here](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts). The TextField interactor extends the [FormField](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts#L6) interactor which is from where it inherits the locator function._

Reviewer: ```suggestion
_See the source code for the TextField interactor [here](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts). The TextField interactor extends the [FormField](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts#L6) interactor, so it has a locator function._
```

Just simplifying some wording here.

> _This is just the way the built-in TextField interactor is configured. It is possible to modify pre-existing interactors such as this TextField interactor or create your own interactors from scratch to suit your needs. We will be going over how you can do all of this on the [Writing Interactors](/docs/interactors/write-your-own) page._

Author: I removed the explicit mention of extend. I think at this point of the docs, people who wants to dive into the source code will do so, and for the others we should keep it more general.

Author: This git diff isn't outdated. Github bug I think.

TextField({ id: 'username-id', placeholder: 'USERNAME', visible: true }).exists();
```

> The filters available are defined by each interactor, so look at the source code for the built-in interactors or the code for your own interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters.

Reviewer: ```suggestion
The filters available are defined by each interactor, so look at the source code for the built-in interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters.
```

I think only pointing to the source code is ok here.

:::note

> Since the TextField interactor extends on the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) which extends from the [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts), the TextField interactor inherits all of the FormField and HTML interactor properties. So in total there are actually eleven filters available for the TextField interactor. 

Reviewer: ```suggestion
There are many filters available for the TextField interactor. The TextField interactor inherits all of the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) and [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts) properties.
```

Once there are more than a few of something, it's best to leave numbers out so that they don't have to be updated when the codebase changes. 

I think it is ok to cut down the explanation some here. If someone doesn't understand inheritance, they would not understand the longer explanation. If they do, then a short explanation should be enough.

Author: Sounds good. But what should we do about the paragraph above it?

> `The filters available are defined by each interactor, so look at the source code for the built-in interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters.`

Should we snip out the last sentence starting with `For example`?

Reviewer: Here's one way you could make these flow together:

> The filters available are defined by each interactor, so look at the source code for the built-in interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters. In addition, TextField inherits all of the properties of the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) and [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts).

We will be exploring this concept more in-depth on the [Writing Interactors](/docs/interactors/write-your-own) page.

> :::

Author: The Assertions page is introduced again in the `Up Next` section only a few paragraphs after here so I removed it and replaced it with this side note to clarify how the filters are being inherited.

It’s that simple! Because Interactors can only match one element, there’s no ambiguity over which submit button you clicked. Additionally, the action will not occur if the Interactor is not there, so you don’t have to worry about checking if the button exists before issuing a click action.

> On the [Writing Interactors](/docs/interactors/write-your-own) page we'll also explain how you can construct your own actions to suit your needs when you create your own Interactors.

Author: I removed this line as it felt repetitive. We reference the `Writing Interactors` page earlier on the same page.
